src/elements/content-uploader/DroppableContentV2.scss

```diff
@@ -0,0 +1,46 @@
+@import '../common/variables';
+@import '../../styles/mixins/responsive';
+
+.bcu-droppable-content-container {
```

Comment:
All new styles should be SUIT based, go to /display/ETO/CSS+Class+Naming on confluence.
For this component use bcu- as the prefix.
We also want to avoid the use of the words "wrapper" or "container" whenever possible.
So for example, this will be .bcu-DroppableContentV2